### DISCUSSION TITLE
Can I disable both archival history and visibility config in development_template.yaml?

### DISCUSSION THREAD

**Discussion by zhangkyou** [uber/cadence#4451] [Q&A]
What does it matters if I disabled both archival history and visibility config in development_template.yaml?

**Comment by longquanzheng** [comment]
Yes. We disabled them all in docker compose by default. See https://github.com/uber/cadence/blob/7ca1886f79bca0da2a845e964500b2285250fec0/docker/config_template.yaml#L241

make sure you disabled them on archival and domainDefaults section in the config

**Reply by zhangkyou** [reply]
If I disable it, does the workflow scheduling or cadence-web UI can be affected?

**Reply by longquanzheng** [reply]
No

**Reply by longquanzheng** [reply]
Archival is an optional feature and it doesn't impact any core functionality: https://cadenceworkflow.io/docs/concepts/archival/
